### ISSUE TITLE
Unable to find configs for Mine Treasures and Mine Treasures not spawning.

### ISSUE THREAD

**Issue by SatanDragon** [Mine-Treasure/mine-treasure#15]
I have had Mine Treasures since about Nov of last year. My players love the plugin. However when we updated to 1.21 I have been unable to find the plug in to make sure it is updated and to get the configs set up as well as the Mine Treasures are not spawning in anymore. I do use pebblehost and that has been no help as well.

**Comment by darkequine** [comment]
I started a new world in 1.21 and the treasures weren't initially spawning for me either. I looked around in the commands and ended up running /function mt:init and it started working.

**Comment by SatanDragon** [comment]
> I started a new world in 1.21 and the treasures weren't initially spawning for me either. I looked around in the commands and ended up running /function mt:init and it started working.

I tried that and it says unknown function

**Comment by darkequine** [comment]
Are you using the mod or the datapack? I don't know if the functions are the same or not.

**Comment by SatanDragon** [comment]
I'm using the Java plug-in so I'm not really sure the difference to be honest

**Comment by darkequine** [comment]
Java plug-in? So the mod? Do you have a .jar file installed?

**Comment by SatanDragon** [comment]
Yes

**Comment by darkequine** [comment]
Weird. You should be able to run the command. With it being on a server though, not sure if everyone who plays needs to also run the command client side vs just on the server.

**Comment by SatanDragon** [comment]
We didn't have to have everyone else run it before. We were able to just get it going. But my IT did it and I have no idea how he did it.

**Comment by darkequine** [comment]
They may have had different permissions to run commands on the server. Glad it's working for you now though.

**Comment by SatanDragon** [comment]
It isn't though....

**Comment by darkequine** [comment]
Hmm. Have you tried restarting the server after the command was run? In my single player world, it started working immediately after I ran the command, but I am unsure on how servers work.

**Comment by darkequine** [comment]
There was a full re-write of the mod because of the changes 1.21 made. Maybe players do need to run the command now as well?

**Comment by SatanDragon** [comment]
Yes. And unfortunately the guy who knew how to do all that decided to up and leave right after the update and I am running with little knowledge.

**Comment by darkequine** [comment]
I don't know PebbleHost but there should be a way to stop then start the server in the panel on the website

**Comment by SatanDragon** [comment]
I have done that. I'm currently running a backup and have to shut it down to do that

**Comment by darkequine** [comment]
Okay. I'm not sure what else to try if running the command locally doesn't work after a server restart.

**Comment by Elias153** [comment]
I agree, the function command is unknown, as well as the scoreboard command. I am running this mod / data pack locally, so that is not an issue. 

I think this issue is quite important since this makes it so players are unable to configure the data pack through these commands.

**Comment by tommytomtj** [comment]
Same issue as @Elias153. I'm running locally w/o cheats enabled so I'm unable to run the /function command. Would it be possible to load settings from a config file?
